services/personality_insights/v3.js

```diff
@@ -15,9 +15,11 @@
  **/
 
 module.exports = function (RED) {
-  const SERVICE_IDENTIFIER = 'personality-insights';
+  const SERVICE_IDENTIFIER = 'personality-insights',
+  PersonalityInsightsV3 = require('ibm-watson/personality-insights/v3'),
+  { IamAuthenticator } = require('ibm-watson/auth');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 2.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)